### PR TITLE
[onert] get output rank and dim from Tensor (not from operand)

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This enables `getOutputOperandDimensions` and `getOutputOperandRank` to get rank and dim from Tensor, not from Operand.

parent issue #56 
draft #52

Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

**Comment by hyunsik-yoon** [comment]
please start reviewing from `ANeuralNetworksExecution.cc` file. That will be easier.

File: runtime/onert/core/include/interp/InterpExecutor.h [hunk 1]
```diff
@@ -59,9 +59,12 @@ class InterpExecutor final : public exec::IExecutor
    */
   void execute(const exec::IODescription &desc) final;
 
+  void fillOutputShapes(std::unordered_map<ir::IOIndex, ir::Shape> *output_shapes) override;
+
 private:
   const ir::Graph &_graph;
   ir::OperandIndexMap<std::shared_ptr<ITensor>> _tensor_map;
+  bool _execution_done;
```
Comment (hseok-oh) [inline runtime/onert/core/include/interp/InterpExecutor.h:67]: `InterpExecutor` is thread-unsafe, so it should not have state variable. 
One solution is to use `IODescription`, or pass by `execute(..)`'s parameter to fill shape result.

**Comment by ragmani** [comment]
Please rebase after merging #1408

File: runtime/onert/core/include/interp/InterpExecutor.h (hunk 1 above)
Comment (hyunsik-yoon) [reply runtime/onert/core/include/interp/InterpExecutor.h:67]: Thanks for the comment.
I think I'd better to use `IODescription` for both `InterpExecutor` and `ExecutorBase`.
I will re-write the code. Thanks!

**Comment by hyunsik-yoon** [comment]
PTAL

**Pull request merged**